README.md

```diff
@@ -418,6 +418,8 @@ Variable | Description | Default value
 `XMPP_INTERNAL_MUC_MODULES` | Custom Prosody modules for internal MUC component (comma separated) | info,alert
 `GLOBAL_MODULES` | Custom prosody modules to load in global configuration (comma separated) | statistics,alert
 `GLOBAL_CONFIG` | Custom configuration string with escaped newlines | foo = bar;\nkey = val;
+`PROSODY_VIRTUALHOST_CONFIG` | Custom configuration string with escaped newlines for jitsi VirtualHost | foo = bar;\nkey = val;
+`PROSODY_MUC_CONFIG` | Custom configuration string with escaped newlines for muc.meet.jitsi Component | foo = bar;\nkey = val;
```

Comment:
These change need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)